sites/cheerp/src/pages/index.astro

```diff
@@ -1,37 +1,45 @@
 ---
 import ProductHome from "@leaningtech/astro-theme/layouts/ProductHome.astro";
 import LinkButton from "@leaningtech/astro-theme/components/LinkButton.astro";
+import heroAsset from "../assets/cheerpcube.png";
+import { Image } from "astro:assets";
 ---
 
-<ProductHome>
-	<section class="py-16">
+<ProductHome style = "px-6 py-10 w-full  mx-auto bg-gradient-to-b to-[#2f8f8a] from-stone-900">
+
+	<section class="px-48 flex col-span-2 items-center">
 		<div class="mx-auto max-w-[56em] text-center space-y-12">
 			<h1
-				class="text-7xl font-semibold text-balance bg-gradient-to-b from-cheerp via-cheerp to-white text-transparent bg-clip-text"
+				class="text-white text-4xl md:text-5xl lg:text-6xl xl:text-7xl font-semibold my-4 text-left"
```

Comment:
Add text-balance to make the text wrap more cleanly.

text-wrap: initial
![image](https://github.com/user-attachments/assets/39a3bfb8-0e76-4221-b521-4edec01dc7ce)

text-wrap: balance
![image](https://github.com/user-attachments/assets/93d7bd3e-1ea0-4816-961a-a5b01712cfae)